.github/workflows/cwag-workflow.yml

```diff
@@ -0,0 +1,369 @@
+name: cwag-workflow
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'orc8r/**'
+      - 'lte/**'
+      - 'feg/**'
+      - 'cwf/**'
+  pull_request:
+    branches:
+      - master
+    paths:
+      - 'orc8r/**'
+      - 'lte/**'
+      - 'feg/**'
+      - 'cwf/**'
+
+jobs:
+
+########## CWAG-PRECOMMIT ##########
+
+  cwag-precommit:
+
+    runs-on: ubuntu-latest
+
+    env:
+      GO111MODULE: on
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Export Magma root path
+      run: echo "MAGMA_ROOT=${PWD}" >> $GITHUB_ENV
```

Comment:
Instead of defining MAGMA_ROOT like this, we should use the github predefined variable

```
    env:
      MAGMA_ROOT: "${{ github.workspace }}"
```